src/Translate.elm

```diff
@@ -279,15 +279,15 @@ translate language appString =
                     }
 
                 WavAStr ->
-                    { en = "When digitising analogue audio, the commonly accepted preservation format is linear pulse code modulation (LPCM) encoding inside a WAVE (.WAV) wrapper. A bit-depth of 24 and sample rate of 96kHz is recommended."
-                    , fr = "Quand on numérise l'audio analogique, le format de conservation bien accepté est l'encodage LPCM (Linear Pulse Code Modulation) à l'intérieur du format conteneur WAVE (.WAV). On recommande une profondeur de 24 bit et une cadence d'échantillonnage de 96 kHz."
-                    , it = "Durante la digitalizzazione dell'audio analogico, il formato di conservazione abitualmente accettato è la codifica LPCM (linear pulse code modulation) all'interno dal formato contenitore WAVE (.WAV). Si consiglia una profondità di 24 bit e una frequenza di campionamento di 96 kHz."
+                    { en = "When digitising analogue audio, the commonly accepted preservation format is linear pulse code modulation (LPCM) encoding inside a WAVE (.WAV) wrapper. The International Association of Sound and Audiovisual Archives (IASA) recommends a bit-depth of 24 and sample rate of 96kHz, but other options are also widely used, such as 24-bit/48kHz and 16-bit/48kHz."
+                    , fr = "Quand on numérise l'audio analogique, le format de conservation bien accepté est l'encodage LPCM (Linear Pulse Code Modulation) à l'intérieur du format conteneur WAVE (.WAV). L'Association Internationale des Archives Sonores et Audiovisuelles (IASA) recommande une profondeur de 24 bit et une cadence d'échantillonnage de 96 kHz, mais autres options sont communément utilisées, e.g. 24 bit/48 kHz et 16 bit/48 kHz."
```

Comment:
L'Association Internationale des Archives sonores [small cap, because adjective _after_ substantive] et Audiovisuelles (IASA) recommande une profondeur de 24 bit et une cadence d'échantillonnage de 96 kHz, mais d'autres [with "d'"] options sont communément utilisées, p.ex. [or even "par exemple" in full, e.g. is not common] 24 bit/48 kHz et 16 bit/48 kHz.

Reply:
Will change the latter two, but in fact IASA capitalises _sonores_! See https://www.iasa-web.org/fr/a-propos-de-iasa, and the general IASA page banner.